pep-0612.rst

```diff
@@ -0,0 +1,647 @@
+PEP: 612
+Title: Parameter Specification Variables
+Author: Mark Mendoza <mendoza.mark.a@gmail.com>
+Sponsor: Guido van Rossum <guido@python.org>
+Discussions-To: Typing-Sig <typing-sig@python.org>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 18-Dec-2019
+Python-Version: 3.9
+Post-History: 18-Dec-2019
+
+Parameter Specification Variables
+=================================
+
+Abstract
+--------
+
+There currently are two ways to specify the type of a callable, the
+``Callable[[T1, T2], TReturn]`` syntax defined in  `PEP 484
```

Comment:
nit: Given that this is just an example, do you want to have concrete types like `int` instead of undefined `T1`, etc.?